wiki/markdown/running-eiffel-intelligence.md

```diff
@@ -2,13 +2,16 @@
 
 ## _Prerequisites_
 
-Eiffel intelligence is Spring microservice distributed in a war file. To start
-the service download the war file from
+Eiffel intelligence is Spring microservice distributed in a war file.
+
+Released Eiffel Interlligence front-end war files can be downloaded from Jitpack:
```

Comment:
Intelligence.
Also consider putting release between 'front-end' and 'war' so it reads like this:
Eiffel Intelligence front-end release war files can be downloaded from Jitpack
Otherwise, since release is a verb, when you start reading the sentence it reads like a request or command to do a release.

Reply:
fixed.